pkg/specgen/generate/config_freebsd.go

```diff
@@ -9,16 +9,22 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/containers/podman/v5/libpod"
 	"github.com/opencontainers/runtime-tools/generate"
 	"github.com/sirupsen/logrus"
 	"golang.org/x/sys/unix"
 	"tags.cncf.io/container-device-interface/pkg/cdi"
 )
 
 // DevicesFromPath computes a list of devices
-func DevicesFromPath(g *generate.Generator, devicePath string) error {
+func DevicesFromPath(g *generate.Generator, devicePath string, rt *libpod.Runtime) error {
 	if isCDIDevice(devicePath) {
+		config, err := rt.GetConfigNoCopy()
+		if err != nil {
+			return fmt.Errorf("getting the Runtime config: %w", err)
+		}
```

Comment:
should should pass in *config.Config here not the full runtime.
Looking at the callers they already have a valid *config.Config so they can just pass that